src/InvokePlaster.ps1

```diff
@@ -830,7 +830,7 @@ function Invoke-Plaster {
                 if (![string]::IsNullOrWhiteSpace($description)) {
                     $newModuleManifestParams['Description'] = $description
                 }
-                if (![string]::IsNullOrWhiteSpace($psVersion)) {
+                if (![string]::IsNullOrWhiteSpace($PowerShellVersion)) {
```

Comment:
No problem.  BTW what you originally had is referred to as `PascalCase` (initial char capitalized) whereas `camelCase` has the initial character lower cased.  I typically use camelCase for local variables and PascalCase for script/globals and function parameters.